bin/run-all-parsers

```diff
@@ -8,14 +8,16 @@
 # It is used to run every test in the suite against each YAML parser.
 #
 #------------------------------------------------------------------------------
+# shellcheck disable=SC2030
+# shellcheck disable=SC2031
 
 set -e -u -o pipefail
 
 main() (
   id=$ID
   tmp=/tmp/test/$id
 
-  set -- $RUNNERS
+  set -- "$RUNNERS"
```

Comment:
Quoting this is almost certainly wrong.
I assume RUNNERS would be a env var list, and this would turn the list into a string.